src/test/java/com/google/api/codegen/testdata/python_doc_setup_library.baseline

```diff
@@ -41,6 +41,7 @@ setup(
     install_requires=install_requires,
     license='Apache-2.0',
     packages=find_packages(),
-    namespace_packages=['google', 'google.cloud', 'google.cloud.gapic', 'google.cloud.gapic.example', 'google.cloud.gapic.example.library'],
+    namespace_packages=['google', 'google.cloud', 'google.cloud.library_v1'],
```

Comment:
`google.cloud.library_v1` is no longer a namespace package. This can be removed from the list.